pkg/services/store/entity/sqlstash/utils.go

```diff
@@ -5,7 +5,17 @@ import (
 	"encoding/hex"
 )
 
-func createContentsHash(contents []byte) string {
-	hash := md5.Sum(contents)
+func createContentsHash(body []byte, meta []byte, status []byte) string {
+	h := md5.New()
+	if meta != nil {
+		_, _ = h.Write(meta)
+	}
+	if body != nil {
+		_, _ = h.Write(body)
+	}
+	if status != nil {
+		_, _ = h.Write(status)
+	}
+	hash := h.Sum(nil)
```

Comment:
There's no need to check for nil afaict:
```suggestion
func createContentsHash(body []byte, meta []byte, status []byte) string {
	h := md5.New()
	_, _ = h.Write(meta)
	_, _ = h.Write(body)
	_, _ = h.Write(status)
	hash := h.Sum(nil)
```